draft-ietf-quic-http.md

```diff
@@ -1024,6 +1025,9 @@ retry some requests.  Including the GOAWAY frame in the same packet as the QUIC
 CONNECTION_CLOSE frame improves the chances of the frame being received by
 clients.
 
+If there are open streams that have not been explicitly closed, they are
+implicitly closed when the connection is closed.
```

Comment:
I appreciate keeping this high level, but I wonder if there's some subtley to stream and connection closure. For instance, https://tools.ietf.org/html/draft-ietf-quic-transport-34#section-10.2 says something like

```
   A CONNECTION_CLOSE frame
   causes all streams to immediately become closed; open streams can be
   assumed to be implicitly reset.

   After sending a CONNECTION_CLOSE frame, an endpoint immediately
   enters the closing state
```

Reply:
Isn't that entirely consistent?

```suggestion
If there are open streams that have not been explicitly closed, they are
implicitly closed when the connection is closed; see Section 10.2 of
{{QUIC-TRANSPORT}}.
```